flink-cdc-e2e-tests/src/test/java/com/ververica/cdc/connectors/e2e/MySqlE2eITCase.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.connectors.e2e;
+
+import com.ververica.cdc.connectors.e2e.utils.FlinkContainerTestEnvironment;
+import com.ververica.cdc.connectors.e2e.utils.JdbcProxy;
+import com.ververica.cdc.connectors.e2e.utils.TestUtils;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.file.Path;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+
+/** End-to-end tests for mysql-cdc connector uber jar. */
+public class MySqlE2eITCase extends FlinkContainerTestEnvironment {
```

Comment:
I think end to end test is different with integration test, how about rename it to `MySqlEndToEndTest` ?
and we use `tests` package prefix in Flink, here we use `e2e`, do we need to keep the same style?

Reply:
There is only unit-test and integration-test phase in `maven-surefire-plugin`, no end-to-end notion in it. We matches `*Test.java` suffix as unit test, and `*ITCase.java` as integration-test. So we have to name it ITCase.

Reply:
It makes sense to me, how about the package prefix? Do you think we need to keep align with Flink project

Reply:
OK. I will use `tests`  package